Meadowlark-js/tests/http/local.ownership-tests.http

```diff
@@ -1,11 +1,42 @@
 ### Set `ACCESS_TOKEN_REQUIRED=true` environment variable before `yarn start:local`
 # Be sure to load all descriptors into the database first.
 
+### Authenticate
```

Comment:
@stevenarnold-ed-fi in this file, we have "expect 403" in several places, inconsistent with `local.token.http` as you've noted. But it makes me wonder if we differentiated between 404 for unauthenticated users and 403 for authenticated users.